elastic/integrations
# Add new fields, delete deprecated ones and  update dashboards for apiserver and controllermanager

## What does this PR do?

Follow up PR to  https://github.com/elastic/beats/issues/31834 and https://github.com/elastic/beats/issues/32007 in which kubernetes `apiserver` and `controllermanager` metricsets where updated to get rid off deprecated fields and collect new ones.

This PR updates the integrations data streams as well and the OOTB dashboards to match those created in the modules.

## Checklist



## How to test this PR locally



Use `elastic-package` to build up the stack and install kubernetes integration with `apiserver` and `controllermanager` enabled. Check the metrics collected and the OOTB dashboards that  all visualisations show data.

## Screenshots



<img width="1753" alt="Controller Manager 1" src="https://user-images.githubusercontent.com/26270880/180781606-cdf9dab0-20f0-48f4-89a6-4613b93e6fe8.png">

<img width="1749" alt="controllermanager 2" src="https://user-images.githubusercontent.com/26270880/180781628-84292bb6-a4bb-4409-95a9-43b67238a2f5.png">

<img width="1749" alt="Apiserver" src="https://user-images.githubusercontent.com/26270880/180781639-f110e391-05e0-4743-adb6-3711dfcd4a45.png">
